_pkgdown.yml

```diff
@@ -44,3 +44,59 @@ articles:
   - new_models
   - report
   - report_table
+
+reference:
+  - title: Report Statistical Information
+    desc: |
+      Main functions for reporting statistical information
```

Comment:
Thanks, I've changed it to for the last two sections. Also wasn't sure whether `report`, `report.default`, `report_table`, and `report_text` should be in the second section (formatting) or first (report statistical info) since they kind of do both. And moving them to the first section would make the second section tiny.